models/stg_customers.sql

```diff
@@ -0,0 +1,7 @@
+ select
+        id as customer_id,
+        first_name,
+        last_name
+
+    from {{ source('jaffle_shop', 'customers') }} 
+    --raw.jaffle_shop.customers
```

Comment:
a pattern we like to follow here, this more stylistic (what you have is fine but this has some benefits) is:

```
with

source as (
  select * from source
),

renamed as (
  select
     id as thing_id,
    status,
    boolen_thing as is_boolean
   etc
)

select * from renamed
```